TowerForge/TowerForge/GameModule/Entities/SpawnableEntities/ArrowTower.swift

```diff
@@ -41,6 +41,9 @@ class ArrowTower: BaseTower, PlayerSpawnable {
         self.addComponent(ShootingComponent(fireRate: ArrowTower.fireRate,
                                             range: ArrowTower.range,
                                             attackPower: ArrowTower.damage, shootingType: Bullet.self))
+        self.addComponent(DamageComponent(attackRate: 1,
```

Comment:
Can replace this with .unit, there's an extension to the Int class with a static constant to avoid magic numbers